recipe/meta.yaml

```diff
@@ -21,13 +21,15 @@ build:
 
 requirements:
   build:
-    - python >=3.5
+    - {{ compiler('c') }}
+  host:
+    - python
     - setuptools
     - pip
   run:
-    - python >=3.5
+    - python
     - setuptools
-    - numpy >=1.13
+    - numpy
```

Comment:
This needs to remain as `>=1.13` as it is a strict requirement for the package to function.